scripts/ci/comment.sh

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env bash
+set -e
+
+# This script posts a comment on a pull request or commit.
+# It first runs `infracost output` to generate the comment and then wraps
+# compost (https://github.com/infracost/compost) to post the comment.
+#
+# The script uses the following environment variables:
+#   - COMMENT_FORMAT: The format of the comment as supported by `infracost output`.
+#       Default: github-comment
+#       Options: github-comment, gitlab-comment
+#   - COMMENT_BEHAVIOR: The behavior of the comment as supported by `compost`.
+#       Default: update
+#       Options:
+#         update: create a single comment and update it. The "quietest" option.
+#         delete-and-new: delete previous comments and create a new one.
+#         hide-and-new: minimize previous comments and create a new one (only supported by GitHub).
+#         new: create a new cost estimate comment on every push.
+#   - COMMENT_TARGET_TYPE: Which objects should be commented on
+#       Default: (empty) - will try and find a pull/merge request, if not it will comment on a commit
+#       Options: pull-request, merge-request, commit.
+#   - COMMENT_TAG: Customize the comment tag.
+#       This is added to the comment as a markdown comment (hidden) to detect
+#       the previously posted comments. This is useful if you have multiple
+#       workflows that post comments to the same pull request or commit.
+#   - COMMENT_PLATFORM: Only required if we need to limit the compost auto-detection to a specific platform.
+#       By default this will be auto-detected.
+#
+# For testing:
+#   - COMMENT_DRY_RUN: Run compost in dry run mode so comments aren't posted, updated or deleted.
+#   - COMMENT_SKIP_COMPOST: Skip the call to compost
+# 
+# Usage:
+#   COMMENT_FORMAT=<FORMAT> COMMENT_BEHAVIOR=<BEHAVIOR> COMMENT_TARGET_TYPE=<TARGET-TYPE> ./comment.sh <INFRACOST_JSON_PATHS>
+#
+# Example:
+#   COMMENT_FORMAT=gitlab-comment COMMENT_BEHAVIOR=update COMMENT_TARGET_TYPE=merge-request ./comment.sh <INFRACOST_JSON_PATHS>
```

Comment:
As it is an example, we probably could also provide the files:
```suggestion
#   COMMENT_FORMAT=gitlab-comment COMMENT_BEHAVIOR=update COMMENT_TARGET_TYPE=merge-request ./comment.sh infracost-file-1.json infracost-file-2.json
```

Reply:
👍